servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -109,7 +109,7 @@ quarkus.log.category."io.quarkus.http.access-log".level=${HTTP_ACCESS_LOG_LEVEL:
 
 ## Quarkus http related settings
 quarkus.http.port=19120
-quarkus.http.test-port=19121
+quarkus.http.test-port=0
```

Comment:
these are the remains of the old port number:
```
~/code/nessie$ rg 19121
clients/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/SparkSqlTestBase.java
56:  protected static final int NESSIE_PORT = Integer.getInteger("quarkus.http.test-port", 19121);

clients/deltalake/src/test/java/org/projectnessie/deltalake/AbstractDeltaTest.java
52:  private static final int NESSIE_PORT = Integer.getInteger("quarkus.http.test-port", 19121);

site/docs/develop/java.md
40:  .withUri(URI.create("http://localhost:19121/api/v1"))
215:  .withUri(URI.create("http://localhost:19121/api/v1"))
224:  .withUri(URI.create("http://localhost:19121/api/v1"))
```

note that spark/delta use the nessie-apprunner where `quarkus.http.test-port` gets injected:
https://github.com/projectnessie/nessie-apprunner/blob/main/apprunner-gradle-plugin/src/main/java/org/projectnessie/quarkus/gradle/QuarkusAppExtension.java

i did let the `quarkus-server` tests print their system properties and `test.url` was the only thing that was set (also when running from intellij)